--- a/bnx2fc/driver/Makefile
+++ b/bnx2fc/driver/Makefile
@@ -349,15 +349,7 @@ bnx2fc_pfc:
 
 bnx2fc_install:
 	mkdir -p $(PREFIX)/lib/modules/$(KVER)/kernel/drivers/scsi;
-ifeq ($(DISTRO), "RHEL")
-	install -m 444 $(BCM_DRV) $(PREFIX)/lib/modules/$(KVER)/extra;
-else
-	@if [ -f $(PREFIX)/lib/modules/$(KVER)/updates ] ; then echo "updates directory present" ;\
-	else mkdir -p $(PREFIX)/lib/modules/$(KVER)/updates ;\
-	fi
-
-	install -m 444 $(BCM_DRV) $(PREFIX)/lib/modules/$(KVER)/updates;
-endif
+	install -m 444 $(BCM_DRV) $(PREFIX)/lib/modules/$(KVER)/override;
 	@if [ "$(PREFIX)" = "" ]; then /sbin/depmod -a ;\
 	else echo " *** Run '/sbin/depmod -a' to update the module database.";\
 	fi
